client/consensus/manual-seal/src/consensus/aura.rs

```diff
@@ -81,7 +80,7 @@ where
 		// we always calculate the new slot number based on the current time-stamp and the slot
 		// duration.
 		let digest_item = <DigestItem as CompatibleDigestItem<AuthoritySignature>>::aura_pre_digest(
-			(time_stamp / self.slot_duration).into(),
+			(time_stamp / self.slot_duration.as_millis() as u64).into(),
```

Comment:
```suggestion
			((time_stamp.as_duration().as_millis() / self.slot_duration.as_millis()) as u64).into(),
```

Reply:
I see this multiple done multiple times in this PR; maybe we should just put it in a method?

Maybe something like this:

```rust
impl Slot {
    pub fn from_timestamp(timestamp: Timestamp, duration: SlotDuration) -> Self {
        ...
    }
}
```